schedulezoomtalks.py

```diff
@@ -0,0 +1,99 @@
+import os
+import github
+import requests
+from researchseminarsdotorg import *
+from requests import get, post, put
+from io import StringIO
+from ruamel.yaml import YAML
+from common import *
+
+TALKS_FILE = "speakers_corner_talks.yml"
+
+def schedule_zoom_talk(talk) -> string:
+
+    # Form the talk registration body
+    request_body =
+    {
+      "topic": "Speakers\' corner talk by %s"%(talk.get("name")),
+      "type": 2, # Scheduled meeting
+      "start_time": talk.get("time"), #2020-03-31T17:00:00
+      "timezone": "UTC",
+      "duration": 60, # 90 minutes
+      "schedule_for": "vsf@virtualscienceforum.org",  # Zoom user ID or Zoom email address
```

Comment:
Correct?

Reply:
No, speakers' corner user (its ID is in common.py as a constant)